DatabindingWithRecyclerView/README.md

```diff
@@ -0,0 +1,5 @@
+# Databinding
+Various examples on databinding which can help not only understand how to get started with it but it also shows how this can be used in complex and real use cases
```

Comment:
The branch looks perfect to me. However it might be better to have a even clear description about the project. So instead of saying various examples, is it possible that we put it in more descriptive way? We'd surely be wanting short description like the current one but maybe there is a even clear way possible. What do you say? Do you also see slight more explanation here?

Reply:
Actually this is your sentence @rajthe innovator :) I just added credits to the generic readme you added in the beginning. But, I agree. We can have more detailed readmes. They can be project specific, describing each sample individually.

Reply:
Yeah, now I remember that. Thanks a lot :+1: But I think I put it that way because the sample idea was not very clear at that time. Now maybe we can add more concrete facts/description. Can you please do that here?

Reply:
Sure. I'll deal with that tomorrow.